tests/replicaset/readconcern-001.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+ReadConcern: MongoDB\Driver\Manager::executeQuery() with readConcern option
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(REPLICASET) ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$bulk = new MongoDB\Driver\BulkWrite();
+$bulk->insert(['_id' => 1, 'x' => 1]);
+$bulk->insert(['_id' => 2, 'x' => 2]);
+$manager->executeBulkWrite(NS, $bulk);
+
+$rc = new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LOCAL);
+$query = new MongoDB\Driver\Query(['x' => 2], ['readConcern' => $rc]);
+$cursor = $manager->executeQuery(NS, $query);
+
+var_dump(iterator_to_array($cursor));
+
+$rc = new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::MAJORITY);
+$query = new MongoDB\Driver\Query(['x' => 2], ['readConcern' => $rc]);
+
+echo throws(function() use ($manager, $query) {
+    $manager->executeQuery(NS, $query);
+}, 'MongoDB\Driver\Exception\RuntimeException'), "\n";
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+array(1) {
+  [0]=>
+  object(stdClass)#%d (%d) {
+    ["_id"]=>
+    int(2)
+    ["x"]=>
+    int(2)
+  }
+}
+OK: Got MongoDB\Driver\Exception\RuntimeException
+Majority read concern requested, but server was not started with --enableMajorityReadConcern.
```

Comment:
Totally my fault. My custom shell script that restarts MO (necessary if some of the cluster tests leave it in a bad state) was missing `--enable-majority-read-concern`. I've got it working now, and have updated the test to insert with a majority write concern so that both the local and majority read concern pick up the data. :+1: